js/src/components/google-combo-account-card/connect-ads/connect-ads-footer.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+
+/**
+ *
+ * @param {Object} props Props.
+ * @param {boolean} props.isLoading Whether the footer is in a loading state.
+ * @param {Function} props.onCreateNew Callback to handle the new ads account creation.
+ * @return {JSX.Element} Footer component.
+ */
+const ConnectAdsFooter = ( { isLoading, onCreateNew = () => {} } ) => (
```

Comment:
```suggestion
const ConnectAdsFooter = ( { isLoading, onCreateNew = noop } ) => (
```
@ankitrox Are we actively using those props? If no, I suggest we remove them for now.